roles/smallfile-bench/tasks/main.yml

```diff
@@ -0,0 +1,50 @@
+---
+- name: Generate smallfile test input yaml
+  k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        namespace: '{{ operator_namespace }}'
+        name: smallfile-test
+      data:
+        smallfilejob: "{{ lookup('template', 'smallfilejob.yaml.j2') }}"
+  when: smallfile.clients > 0
+
+- name: Start Run(s)
+  k8s:
+    definition:
+      kind: Job
```

Comment:
I would recommend we create a template from this jobfile, so we can easily manipulate it -- ie add a pvc eventually

Reply:
There is a template created in roles/smallfile-bench/templates/ , which is further served as yaml input to smallfile

Reply:
Also, yes, I went through fio-bench's tasks/main.yaml, I saw that there were tasks to create a PVC and add it to the container. I can copy the same procedure here.